src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25473.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if !MACCATALYST
+using Microsoft.Maui.Platform;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25473 : _IssuesUITest
+	{
+		public Issue25473(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "MAUI Entry in Windows always shows ClearButton despite ClearButtonVisibility set to 'Never'";
+
+		[Test]
+		[Category(UITestCategories.Entry)]
+		public void VerifyEntryClearButtonVisibilitySetToNever()
```

Comment:
The Windows snapshot is pending.
![image](https://github.com/user-attachments/assets/bae15bbe-3278-44a2-a3de-1eb03202aca1)

Reply:
I hsve added the windows snapshot.